src/depends/jsonrpccpp/common/procedure.cpp

```diff
@@ -13,6 +13,10 @@
 #include <cstdarg>
 #include <vector>
 
+#ifdef __clang__
```

Comment:
Just one point: please check we're allowed to change the source directly here based on the license.